Docs/source/pages/get-started.md

```diff
@@ -486,6 +486,76 @@ AltTester® package contains AltDriver class used to connect to the instrumented
                     :language: py
 
+    .. tab:: robot
+
+        **Prerequisite:**
+            - `Robot Framework <https://robotframework.org/?tab=1#getting-started>`_ installed.
+            - Instrumented build.
+            - The AltTester® Desktop installed on your PC.
+
+        **Connection:**
+            Starting with version ``2.0.0``, the AltTester® Desktop must be running on your PC while the tests are running.
+
+            1. The AltTester® Desktop app is running.
+            2. The instrumented build is running.
+            3. Configure the IP of the build in order to match the IP of the machine (PC) the Desktop app is running on.
+            4. Then press the restart button in the AltTester® dialog and wait until the `Connected to AltTester® Server…` message is displayed in the dialog.
+
+        **Test Setup:**
+            - You can use any IDE you want. As a suggestion, we prefer `Visual Studio Code <https://code.visualstudio.com/>`_  and `PyCharm <https://www.jetbrains.com/pycharm/download/>`_.
+
+        **Writing tests:**
+            - First you need to create a new project with th Robot class under it, which can store your tests
+            - In the ``Settings`` section:
+                - Import the ``AltTesterLibrary``
+                - Next create the ``Setup`` and ``Teardown`` suites for your tests
+                    - In the ``Setup`` suite you need to include the initialization of the AltDriver - make sure to also add the ip, host and app name as parameters if the connection data for the connected instrumented app is not the default one
+                    - In the ``Teardown`` suite the AltDriver needs to be stopped                
+            .. code-block:: robot
+
+                *** Settings***
+                Library    AltTesterLibrary
+                Suite Setup    Initialize Altdriver
+                Suite Teardown    Stop Altdriver 
+            - The ``Test Cases`` section stores all your tests who should have a specific and relevant name             
+            .. code-block:: robot
+
+                *** Test Cases ***
+                Test My First Test
+
+        **Running the tests:**
+            
+            - To run your test use the ``robot`` commmand            
+            - Robot Framework test cases are created in files and directories, and they are executed by giving the path to the file or directory in question to the selected runner script
+            - Examples:
+            .. code-block:: console
+
+                robot my_first_test.robot
+
+            .. code-block:: console
+
+                robot path/to/my_tests/
+
+            - After running the test the Robot will generate 3 files - the ``report.html`` and ``log.html`` files can be opened in the browser and for every test in ``log.html`` there will be a status highlight, the keywords used, as well as a potential error that might have occured during the test
```

Comment:
Maybe I am missing something here, but which are the 3 files that we are talking about?

Reply:
`log.html`, `report.html`, `output.xml`

Also, I took this information from Sandy's demo video.

Reply:
Ok, but is the "output.xml mentioned here, in doc?"

Reply:
no, but I will mention it to avoid confusion